_apprenticeship/links.md

```diff
@@ -38,3 +38,9 @@ slug: further-info
 * [Christian Rackerseder](https://twitter.com/CallistoShip)
 * Periklis Tsirakidis
 * [Sergii Paryzhskyi](https://github.com/HeeL)
+
+<h3>Retired Mentor('s twitter accounts)</h3>
```

Comment:
Maybe I'm looking too far in the future, but when I retire there will be no twitter account but github instead ;)

Reply:
Not sure if this has been done already, but I would ask everyone for permission before I would mention them here.

Reply:
I'd expect them not to mind since they already approved to be mentioned in general, and this is just a section shift.

@gilligan @Sebruck please let me know if you have concerns with being part of the "mentor's hall of fame". We can always remove you afterwards.